modin/pandas/test/test_series.py

```diff
@@ -1580,7 +1577,7 @@ def test_matmul(data):
 
     # Test when input series index doesn't line up with columns
     with pytest.raises(ValueError):
-        modin_result = modin_series @ pd.Series(
+        _ = modin_series @ pd.Series(
```

Comment:
ditto-- why assign? here and below

Reply:
Thought it was better for the linter, but no, he doesn't care. fixed